server/src/main/java/com/broadcom/lsp/cobol/core/model/CobolLine.java

```diff
@@ -84,4 +88,35 @@ public void setSuccessor(final CobolLine successor) {
   public String toString() {
     return sequenceArea + indicatorArea + contentAreaA + contentAreaB + commentArea;
   }
+
+  @Override
+  public Iterator<CobolLine> iterator() {
+    return new CobolLineIterator(this);
+  }
+
+  private class CobolLineIterator implements Iterator<CobolLine> {
```

Comment:
I may be wrong here, but this class explicitly receives the `this` reference from the outer class, so it may be marked `static` to be nested instead of inner

Reply:
Done